code/api/src/seeders/1-user.js

```diff
@@ -22,6 +22,15 @@ module.exports = {
         role: params.user.roles.user,
         createdAt: new Date(),
         updatedAt: new Date()
+      },
+      {
+        name: 'A Stylish User',
+        email: 'stylegirl@crate.com',
+        password: bcrypt.hashSync('123456', config.saltRounds),
+        role: params.user.roles.user,
+        style: 'classic',
```

Comment:
Thanks for adding this user! This will be helpful for the frontend to check that our logic works correctly when a user has an existing style.

Reply:
Woohoo!